models/containerized_model/README.md

```diff
@@ -185,3 +185,56 @@ This should create a docker container and host the webservice.
 
 ### Step 6 - Test
 Make a request to `http://127.0.0.1:8551/predict` with the respective parameters.
+
+
+## [Proxy Template](#proxy-template)
+### Step 1 - Template generation
+
+Generate the code template for containerization of your model:
+```
+./generate.sh -d generated-container-proxy -i certifai-proxy-container:latest -m proxy -n containerproxy -r container-proxy
```

Comment:
let's leave -n and -r off for now & keep this simpler. We dont doc them in --help, and its somewhat unlikely the person doing this step will know what namespace to choose (they may not be k8s aware).   Also suggest dropping -d as in the others. Its easily renamed.

Reply:
fixed